more/problem-sets-competitive-programming.md

```diff
@@ -84,6 +84,7 @@
 
 ### Problem Sets
 
+* [100 Days CSS Challenge] (https://100dayscss.com) - Matthias Martin *(Codepen account requested, not required)*
```

Comment:
```suggestion
* [100 Days CSS Challenge](https://100dayscss.com) - Matthias Martin *(Codepen account requested, not required)*
```
Why does it return day 49?

Reply:
It's on a daily loop and restarts with day one after day 100, https://100dayscss.com is the main page, but other challenges can be accessed by using the arrows 

![image](https://github.com/EbookFoundation/free-programming-books/assets/93568245/e69ab60c-8928-4d76-bc9c-590fa136af5a)

should I link to the 1st day instead? https://100dayscss.com/days/1/

Reply:
It was a bit puzzling, but I think it's best to defer to the site owner as to how to greet new users.